bpf/bpf_lxc.c

```diff
@@ -934,9 +962,13 @@ ipv4_policy(struct __sk_buff *skb, int ifindex, __u32 src_label, int *forwarding
 			return ret2;
 	}
 
-	verdict = policy_can_access_ingress(skb, src_label, tuple.dport,
-					    tuple.nexthdr, sizeof(orig_sip),
-					    &orig_sip, is_fragment);
+	if (!(cfg->flags & EP_F_SKIP_POLICY_INGRESS))
+		verdict = policy_can_access_ingress(skb, src_label, tuple.dport,
+						    tuple.nexthdr,
+						    sizeof(orig_sip),
+						    &orig_sip, is_fragment);
+	//else
+	//	verdict = 0;
```

Comment:
Is this deliberately omitted? Is it because we can trust the verdict to be 0 if we don't run the policy access function? In that case we can likely remove the commented else lines here, and also make equivalent changes in the corresponding ipv6 case so that the code is logically the same for both cases.

Reply:
This was leftover from me trying to reduce complexity (and failing). I'll uncomment it.